python/xorbits/_mars/serialization/serializables/core.py

```diff
@@ -81,14 +81,7 @@ def __new__(mcs, name: str, bases: Tuple[Type], properties: Dict):
                 properties_without_fields[k] = v
                 continue
 
-            field = all_fields.get(k)
-            if field is None:
-                properties_field_slot_names.append(k)
-            else:
-                v.name = field.name
-                v.get = field.get
-                v.set = field.set
-                v.__delete__ = field.__delete__
+            properties_field_slot_names.append(k)
```

Comment:
How is this change related to the coverage?

Reply:
This is a bug found during development.